react/features/base/aspect-ratio/components/AspectRatioDetector.native.js

```diff
@@ -0,0 +1,74 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { calculateNewAspectRatio } from '../actions';
+import styles from './styles';
+
+/**
+ * A root {@link View} which captures the 'onLayout' event and figures out
+ * the aspect ratio of the app.
+ */
+class AspectRatioDetector extends Component {
+    /**
+     * AspectRatioDetector component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The "onLayout" handler.
+         */
+        _onLayout: PropTypes.func,
+
+        /**
+         * Any nested components.
+         */
+        children: PropTypes.object
+    };
+
+    /**
+     * Renders the root view and it's children.
+     *
+     * @returns {Component}
+     */
+    render() {
+        return (
+            <View
+                onLayout = { this.props._onLayout }
+                style = { styles.aspectRatioDetectorStyle } >
+                {this.props.children}
```

Comment:
style: space after { and before } 

Sidenote: I think eslint doesn't catch these because it's not an object literal.